storage-resize-images/package.json

```diff
@@ -14,9 +14,8 @@
   "dependencies": {
     "firebase-admin": "^8.0.0",
     "firebase-functions": "^3.3.0",
-    "mkdirp": "^0.5.1",
-    "mkdirp-promise": "^4.0.0",
-    "sharp": "0.23.4"
+    "mkdirp": "^1.0.4",
+    "sharp": "^0.24.1"
```

Comment:
Sorry, I didn't spot this earlier. The `sharp` library needs fixing to this version `"sharp": "0.23.4"` to ensure compatibility with Node version 8. See this [PR](https://github.com/firebase/extensions/pull/278).